libs/dh/admin/data-access-api/src/lib/dh-admin-user-management-data-access-api.store.ts

```diff
@@ -122,24 +136,22 @@ export class DhAdminUserManagementDataAccessApiStore
           users: [],
         });
       }),
-      switchMap((fetchUsersParams) =>
-        this.getUsers(fetchUsersParams).pipe(
-          tapResponse(
-            (response) => {
+      switchMap((fetchUsersParams) => {
+        return this.getUsers(fetchUsersParams).pipe(
+          tap((response) => {
```

Comment:
suggestion: add `catchError` either here (after the `tap`) or as part of `getUsers`. The `tapResponse` operator handled the errors for us before (e.g. 404, 500) but now we'll need to do it. Or maybe we can still use `tapResponse`. I don't think it's tied to NgRx anymore.